standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java

```diff
@@ -732,10 +756,26 @@ public static void startMetaStore(int port, HadoopThriftAuthBridge bridge,
     }
     // optionally create and start the property server and servlet
     propertyServer = PropertyServlet.startServer(conf);
+    // optionally create and start the Iceberg REST server and servlet
+    icebergServer = startIcebergCatalog(conf);
```

Comment:
Can we able to merge the `propertyServer` and the `icebergServer`, listening to the same port?

Reply:
For security reasons, it might be preferable to have 2 different ports so that routing can filter clients.
Besides, the property server can be configured not to start. (does not start by default I think).

Reply:
Can this be achieved via `hive.metastore.catalog.servlet.path` and `hive.metastore.properties.servlet.path`?

Reply:
It would make sense to host only one Jetty and create connectors/servlets per feature. Albeit this requires even more changes (and some logic to start the server if any of the feature servlets is required) that are not required for the task at hand. This seems like a 'lateral' move wrt handing Iceberg Catalog.